$OpenBSD: patch-testsuite_mk_test_mk,v 1.10 2019/09/30 11:44:18 kili Exp $

Fix detection of available library ways in case the library-dirs:
field of ghc-prim contains more than one entry (only checks file
in the first entry, which is buggy, the whole detection mechanism
looks buggy).

Index: testsuite/mk/test.mk
--- testsuite/mk/test.mk.orig
+++ testsuite/mk/test.mk
@@ -93,7 +93,7 @@ else
 RUNTEST_OPTS += -e ghc_with_native_codegen=False
 endif
 
-GHC_PRIM_LIBDIR := $(subst library-dirs: ,,$(shell "$(GHC_PKG)" field ghc-prim library-dirs --simple-output))
+GHC_PRIM_LIBDIR := $(subst library-dirs: ,,$(shell "$(GHC_PKG)" field ghc-prim library-dirs --simple-output | sed -e 's/[[:space:]].*//'))
 HAVE_VANILLA := $(shell if [ -f $(subst \,/,$(GHC_PRIM_LIBDIR))/GHC/PrimopWrappers.hi ]; then echo YES; else echo NO; fi)
 HAVE_DYNAMIC := $(shell if [ -f $(subst \,/,$(GHC_PRIM_LIBDIR))/GHC/PrimopWrappers.dyn_hi ]; then echo YES; else echo NO; fi)
 HAVE_PROFILING := $(shell if [ -f $(subst \,/,$(GHC_PRIM_LIBDIR))/GHC/PrimopWrappers.p_hi ]; then echo YES; else echo NO; fi)
